secant/Features/App/App.swift

```diff
@@ -117,13 +119,20 @@ extension AppReducer {
                     return Effect(value: .initializeApp)
                 }
             } catch CocoaError.fileNoSuchFile, CocoaError.fileReadNoSuchFile {
-                state.appInitializationState = .filesMissing
+                if keysPresent {
```

Comment:
Could you add comments on why this "error" would be a valid state that is not an error at first sight